packages/eslint-plugin/src/util/createRule.ts

```diff
@@ -1,10 +1,5 @@
 import { ESLintUtils } from '@typescript-eslint/experimental-utils';
 
-// note - cannot migrate this to an import statement because it will make TSC copy the package.json to the dist folder
-// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-const version: string = require('../../package.json').version;
-
 export const createRule = ESLintUtils.RuleCreator(
-  name =>
-    `https://github.com/typescript-eslint/typescript-eslint/blob/v${version}/packages/eslint-plugin/docs/rules/${name}.md`,
+  name => `https://typescript-eslint.io/rules/${name}`,
```

Comment:
@bradzacher I have a vague memory of you mentioning it's preferable to link to the specific version of a rule, but can't find where that was. Did that opinion ever exist, and if so, is it still accurate now that the website is ready?

Reply:
back in the beginning it was a huge problem for us changing rules regularly.
Now-a-days everything is super stable so I don't think it's worth worrying about any more.

If we want to build it into the website - we totally could, but I think at this stage of the project it's probably super low value.